File: Cardiumm-cs50-2018-x-cash-20180816T123752Z/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float cash;
    // Counter to count the coin amount
    int counter = 0;

    // Getting users cash in a positive amount.
    do
    {
        cash = get_float("Your current cash: ");
    }
    while (cash < 0);

    //Turning cash to cents.
    int cents = round(cash * 100);

    //Calculating coins below.
    while (cents >= 25)
    {
        cents -= 25;
        counter++;
        if (cents < 25)
        {
            cents = cents % 25;
        }
    }

    while (cents >= 10)
    {
        cents -= 10;
        counter++;
        if (cents < 10)
        {
            cents = cents % 10;
        }
    }

    while (cents >= 5)
    {
        cents -= 5;
        counter++;
        if (cents < 5)
        {
            cents = cents % 5;
        }
    }


    // Just adding cents / 1 because cents wont be less than 1
    while (cents >= 1)
    {
        counter += cents / 1;
        cents = cents % 1 ;
    }

    printf("%i", counter);
}

File: Cardiumm-cs50-2018-x-sentimental-cash-20180824T214058Z/cash.py
from cs50 import get_float

# Get users cash
while True:
    cash = get_float("Your current cash: ")
    if cash > 0:
        break

# Turn it into cents.
cents = round(cash * 100)
counter = 0

# Calculate the coins
while cents >= 25:
    cents -= 25
    counter += 1
    if cents < 25:
        cents = cents % 25

while cents >= 10:
    cents -= 10
    counter += 1
    if cents < 10:
        cents = cents % 10

while cents >= 5:
    cents -= 5
    counter += 1
    if cents < 5:
        cents = cents % 5

while cents >= 1:
    cents -= 1
    counter += 1
    if cents < 1:
        cents = cents % 1

print(counter)